Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.Droid/Contoso.Forms.Demo.Droid.csproj

```diff
@@ -51,12 +51,12 @@
     <Reference Include="Mono.Android" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AppCenter.Analytics" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AppCenter.Crashes" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AppCenter.Push" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AppCenter.Auth" Version="2.0.0" />
-    <PackageReference Include="Microsoft.AppCenter.Data" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AppCenter.Analytics" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AppCenter.Crashes" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AppCenter.Push" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AppCenter.Auth" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AppCenter.Data" Version="2.1.0" />
     <PackageReference Include="Xamarin.Forms">
       <Version>3.6.0.344457</Version>
```

Comment:
Please update Xamarin.Forms manually in all projects of the Demo.sln, this is also part of the release procedure.

Reply:
Versions have been updated.